manifest.json

```diff
@@ -3787,6 +3834,75 @@
                 }
             }
         },
+        "Banane9.CorrectGizmoScaling": {
+            "name": "CorrectGizmoScaling",
+            "description": "Tiny mod that corrects the scaling of the Inspector spawned by the MaterialTip's Edit Material context menu button.",
+            "category": "Developers",
+            "website": "https://github.com/Banane9/NeosCorrectGizmoScaling",
+            "sourceLocation": "https://github.com/Banane9/NeosCorrectGizmoScaling",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/Banane9/NeosCorrectGizmoScaling/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosCorrectGizmoScaling/releases/download/v1.0.0/CorrectGizmoScaling.dll",
+                            "sha256": "749B222F8FE36B10985E2AA1076E649CA72F0EF1B2D311D00FC7DF3D83C8D031"
+                        }
+                    ]
+                }
+            }
+        },
+        "Banane9.BoundedUIX": {
+            "name": "BoundedUIX",
+            "description": "Makes UIX elements selectable and editable with the usual Slot gizmos, as well as fixing UIX elements breaking when a parent is inserted.",
+            "category": "Developers",
+            "website": "https://github.com/Banane9/NeosBoundedUIX",
+            "sourceLocation": "https://github.com/Banane9/NeosBoundedUIX",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "2.2.2": {
+                    "releaseUrl": "https://github.com/Banane9/NeosBoundedUIX/releases/tag/v2.2.2",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/Banane9/NeosBoundedUIX/releases/download/v2.2.2/BoundedUIX.dll",
+                            "sha256": "7B2FEEBC1A6EB82AC65455AFB060C2DA7328B3E651805E4A2EEFB3DA210140F1"
+                        }
+                    ]
+                }
+            }
+        },
+        "Banane9.ComponentAttacherSearch": {
+            "name": "ComponentAttacherSearch",
+            "description": "Adds a search bar to component attachers.",
```

Comment:
May want to either include some of this in the description or add your [ComponentAttacherFavorites](https://github.com/Banane9/NeosComponentAttacherFavorites) as a requirement.

 "Requires my forked version of [ComponentAttacherFavorites](https://github.com/Banane9/NeosComponentAttacherFavorites) to work ideally.
Workaround is to make the this mod load before the other."

Reply:
Added my favorites version to the list as well with a note in the description of the search about which one to use / what to do.